db/migrate/20210806114723_create_cinema_halls.rb

```diff
@@ -4,7 +4,7 @@ class CreateCinemaHalls < ActiveRecord::Migration[6.1]
   def change
     create_table :cinema_halls do |t|
       t.string :name, null: false
-      t.string :row, null: false
+      t.integer :row_number, null: false
```

Comment:
As you already merged this migration, you should not edit it - if you have to change field name then it would be best to create migration that changes this field name.

Reply:
I roll backed this change and will create a separate migration, thanks!